sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql

```diff
@@ -0,0 +1,46 @@
+-- create table & idx
+create table rvc_tmp2(col1 int, col2 int, col3 int, col4 int, col5 int);
+insert into rvc_tmp2 values(1,2,3,4,5),(1,1,1,1,1),(2,2,2,2,2),(3,3,3,3,3);
+create index idx on rvc_tmp2(col1,col2,col3,col4);
+create index idx2 on rvc_tmp2(nvl(col1,1),col2,col3,col4);
+
+-- check for query plan
+set optimization level 513;
+
+select /*+ recompile */ * from rvc_tmp2 where (col1,col2) IN ((1,2),(2,2)) and col3 = 3 and col4 = 4;
```

Comment:
How about adding Test case classification such as 'combination of predicates', 'combination of predicates with join'.

Reply:
good idea! I will next refer to this.